FLEDGE.md

```diff
@@ -646,11 +646,12 @@ The arguments to `generateBid()` are:
       'joinCount': 3,
       'recency': 3600000,
       'bidCount': 17,
-      'prevWins': [[time1,ad1],[time2,ad2],...],
+      'prevWins': [{'timeDelta': time1, 'adJSON': prevWinningAd1JSONString}, {'timeDelta': time2, 'adJSON': prevWinningAd2JSONString}, ...],
       'wasmHelper': ... /* a WebAssembly.Module object based on interest group's biddingWasmHelperURL */
       'dataVersion': 1, /* Data-Version value from the trusted bidding signals server's response(s) */
     }
     ```
+    The `prevWins` is a list of [`PreviousWin` dictionaries](https://wicg.github.io/turtledove/#dictdef-previouswin), for all of the Interest Groups previous wins on the browser. `timeDelta` is the difference between the auction start time and the time of the previous win. `adJSON` is the stringified JSON of what was returned in the `ad` element by `generateBid` for that Interest Group's winning bid in that auction. 
```

Comment:
```suggestion
```

Reply:
Shouldn't we update this paragraph to be prevWinsMs, instead of removing it?